introspection-engine/introspection-engine-tests/tests/tables/mod.rs

```diff
@@ -640,19 +671,60 @@ async fn negative_default_values_should_work(api: &TestApi) -> crate::TestResult
         )
         .await?;
 
-    let dm = indoc! {r##"
-        model Blog {
+    let float_native = if api.sql_family().is_mysql() {
+        "@db.Float"
+    } else if api.sql_family().is_mssql() {
+        "@db.Real"
+    } else {
+        ""
+    };
+
+    let dm = formatdoc! {r##"
+        model Blog {{
           id                     Int     @id @default(autoincrement())
           int                    Int     @default(1)
           neg_int                Int     @default(-1)
-          float                  Float   @default(2.1)
-          neg_float              Float   @default(-2.1)
+          float                  Float   @default(2.1) {}
+          neg_float              Float   @default(-2.1) {}
           big_int                BigInt  @default(3)
           neg_big_int            BigInt  @default(-3)
-        }
-    "##};
+        }}
+    "##, float_native, float_native};
 
-    assert_eq_datamodels!(dm, &api.introspect().await?);
+    api.assert_eq_datamodels(&dm, &api.introspect().await?);
 
     Ok(())
 }
+
```

Comment:
This was for the bug I initially intended to fix. Will pick this up after merging.